workflow-runtime/src/main/java/com/squareup/workflow/internal/RealRenderContext.kt

```diff
@@ -24,7 +24,7 @@ import com.squareup.workflow.Workflow
 import com.squareup.workflow.WorkflowAction
 import kotlinx.coroutines.channels.SendChannel
 
-class RealRenderContext<out PropsT, StateT, OutputT>(
+internal class RealRenderContext<out PropsT, StateT, OutputT>(
```

Comment:
Wow.

Reply:
ABI golf: what's the smallest change you can make to reduce your public ABI as much as possible.